cachy-chroot/PKGBUILD

```diff
@@ -0,0 +1,43 @@
+# Maintainer: SoulHarsh007 <harsh.peshwani@outlook.com>
+
+pkgname=cachy-chroot
+pkgver=1.0.0
+pkgrel=1
+pkgdesc='Chroot helper for CachyOS'
+arch=('x86_64' 'x86_64_v3')
```

Comment:
I think we can remove this. We are using nowadays --ignorearch or overwrite it with pacman PACKAGEARCH feature

Reply:
Its anyways cannot be build with v3 carch